services/coderabbit/coderabbit-stats.tester.js

```diff
@@ -0,0 +1,32 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('live CodeRabbitStats')
+  .get('/prs/github/coderabbitai/ast-grep-essentials.json')
+  .expectBadge({
+    label: 'coderabbit reviews',
+    message: Joi.number().min(0),
+  })
+
+t.create('live CodeRabbitStats demo repo')
+  .get('/prs/github/coderabbitai/coderabbit-docs.json')
+  .expectBadge({
+    label: 'coderabbit reviews',
+    message: Joi.number().min(0),
+  })
+
```

Comment:
What's the benefit of this test? how does it differ from `live CodeRabbitStats`?
```suggestion

```

Reply:
Updated the test file.